server/game/core/card/Card.ts

```diff
@@ -738,6 +738,19 @@ export class Card extends OngoingEffectSource {
         const isActivePlayer = activePlayer === this.controller;
         const selectionState = activePlayer.getCardSelectionState(this);
 
+        // add exhausted state if the card can be exhausted and is in play
+        let exhaustedState = {};
+        if (this.canBeExhausted() && this.canBeInPlay() && ((this as unknown as InPlayCard).isInPlay() || this.location === Location.Resource)) {
```

Comment:
Probably better to override `getSummary()` in `PlayableOrDeployableCard` and have it append the exhausted property to the state. That way we don't need to jump through all these hoops here to see if the property exists or not, if we do it from `PlayableOrDeployableCard` it has direct access to the `_exhausted` field and can just drop its value right in. If it's in an invalid location for being exhausted, the value will just be null.

Separately, as a tip for future use (it's not relevant here if we make the above change), you shouldn't need to make the cast of `this` to `InPlayCard` in this situation. `this.canBeInPlay()` has return type `this is InPlayCard` which means that after `this.canBeInPlay()` is determined to be true, typescript will assume that `this` has all of the properties of `InPlayCard`.